### PR TITLE
remove old css-loader

### PR DESCRIPTION
**Pull request by rianfowler**
## Description
- Removes the staging flag for new version of `css-loader` which changes css minification on va.gov 
- [details](https://github.com/department-of-veterans-affairs/va.gov-team/issues/15178#issuecomment-714636751) about breaking changes updates <- **Please review this for accuracy**
- [info](https://github.com/department-of-veterans-affairs/vets-website/pull/14685#issue-508565369) on diff

## Testing done
- the new css minifier is used on staging. spot checked IE 11 on facility locator, GIBCT, preneed form, crisis modal, homepage, and a few content pages. 

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: config/webpack.config.js [hunk 1]
```diff
@@ -139,28 +139,18 @@ module.exports = env => {
...
-              ? {
-                  loader: 'css-loader-1.0.0',
-                  options: {
-                    sourceMap: enableCSSSourcemaps,
-                  },
-                }
-              : {
-                  loader: 'css-loader-0.28.10',
-                  options: {
-                    minimize: isOptimizedBuild,
```
Comment (rianfowler) [inline config/webpack.config.js:152]: Minimization was removed from `css-loader` so this follows the [CHANGELOG](https://github.com/webpack-contrib/css-loader/blob/master/CHANGELOG.md#100-2018-07-06) recommendation to use `cssnano` in `postcss-loader` for css minification.

File: config/webpack.config.js [hunk 2]
```diff
@@ -139,28 +139,19 @@ module.exports = env => {
...
               loader: 'postcss-loader',
               options: {
-                plugins:
-                  buildOptions.buildtype === ENVIRONMENTS.VAGOVSTAGING
-                    ? // TODO: check if isOptimizedBuild when ready for production
-                      () => [require('autoprefixer'), require('cssnano')]
-                    : () => [require('autoprefixer')],
+                // use cssnano to minimize css only on optimized builds
+                plugins: isOptimizedBuild
+                  ? () => [require('autoprefixer'), require('cssnano')]
```
Comment (rianfowler) [inline config/webpack.config.js:153]: `isOptimizedBuild` moved to `postcss-loader` from old `css-loader`

**Review by jhonny218 — APPROVED** [review]

**Review by mdewey — APPROVED** [review]

**Review by erikphansen — APPROVED** [review]

**Review by micahchiang — APPROVED** [review]

**Review by Mottie — APPROVED** [review]

**Review by brangi — APPROVED** [review]

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by bradl3yC — APPROVED** [review]

**Review by craigwheeler — APPROVED** [review]

**Review by LeKeve — APPROVED** [review]

**Comment by rianfowler** [comment]
Just a reminder, we're merging this in tomorrow.

**Pull request merged**